eclipse-test/swtbot-eclipse-test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,32 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Eclipse Plug-In Tests
+Bundle-SymbolicName: devonfw-swtbot-eclipse-test;singleton:=true
+Bundle-Version: 3.2.1
+Bundle-Vendor: Capgemini
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .,
+ lib/slf4j-api-1.7.21.jar,
+ lib/assertj-core-3.8.0.jar,
+ lib/core-test-5.3.1.jar,
+ lib/jcl-over-slf4j-1.7.21.jar,
+ lib/log4j-over-slf4j-1.7.21.jar,
+ lib/logback-classic-1.1.2.jar,
+ lib/logback-core-1.1.2.jar
```

Comment:
The Bundle classpath seems to be out of sync with the build.properties. Please make sure, that your code is portable such that everybody can execute it.

Reply:
For eclipse bundle projects, you have to adapt the plugin.xml to specify the "runtime dependencies"